front/src/routes/integration/all/zigbee2mqtt/setup-page/SetupTab.jsx

```diff
@@ -0,0 +1,262 @@
+                  <th>
+                    <Text id="systemSettings.containers" />
+                  </th>
+                  <th>{`Status`}</th>
+                </thead>
+                <tbody class="text-center">
+                  <tr>
+                    <td>{`Gladys`}</td>
+                    <td>
+                      <span class="tag tag-success">
+                        <Text id={`systemSettings.containerState.running`} />
+                      </span>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>{`MQTT`}</td>
+                    <td>
+                      {props.mqttRunning && (
+                        <span class="tag tag-success">
+                          <Text id={`systemSettings.containerState.running`} />
+                        </span>
+                      )}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>{`Zigbee2mqtt`}</td>
+                    <td>
+                      {props.zigbee2mqttRunning && (
+                        <span class="tag tag-success">
+                          <Text id={`systemSettings.containerState.running`} />
+                        </span>
+                      )}
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+          </div>
+          <div class="card-header d-sm-none">
+            <h2 class="card-title">
+              <Text id="integration.zigbee2mqtt.setup.serviceStatus" />
+            </h2>
+          </div>
+          <div class="row justify-content-center d-sm-none">
+            <div class="col-auto">
+              <table class="table table-responsive table-borderless table-sm">
+                <thead class="text-center">
+                  <th>
+                    <Text id="integration.zigbee2mqtt.setup.link" />
+                  </th>
+                  <th>{`Status`}</th>
+                </thead>
+                <tbody class="text-center">
+                  <tr>
+                    <td>{`Gladys - MQTT`}</td>
+                    <td>
+                      {props.zigbee2mqttExist && props.zigbee2mqttConnected && (
+                        <i style={{ color: '#0f0', fontSize: '24px' }} class="fe fe-check" />
+                      )}
+                      {props.zigbee2mqttExist && !props.zigbee2mqttConnected && (
+                        <i style={{ color: '#f00', fontSize: '24px' }} class="fe fe-x" />
+                      )}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td>{`MQTT - Zigbee`}</td>
+                    <td>
+                      {props.zigbee2mqttExist && props.zigbee2mqttConnected && (
+                        <i style={{ color: '#0f0', fontSize: '24px' }} class="fe fe-check" />
+                      )}
+                      {props.zigbee2mqttExist && !props.zigbee2mqttConnected && (
```

Comment:
Please prefer CSS classes (and classnames import).

Reply:
Fixed